osu.Game.Rulesets.Catch/Objects/Drawables/DrawableCaughtObject.cs

```diff
@@ -41,23 +34,26 @@ protected CaughtObject(CatchSkinComponents skinComponent, Func<ISkinComponent, D
             Size = new Vector2(CatchHitObject.OBJECT_RADIUS * 2);
         }
 
-        /// <summary>
-        /// Copies the hit object visual state from another <see cref="IHasCatchObjectState"/> object.
-        /// </summary>
-        public virtual void CopyStateFrom(IHasCatchObjectState objectState)
+        public virtual void Apply(CaughtObjectEntry entry)
         {
-            HitObject = objectState.HitObject;
-            Scale = Vector2.Divide(objectState.DisplaySize, Size);
-            Rotation = objectState.DisplayRotation;
-            AccentColour.Value = objectState.AccentColour.Value;
-            HyperDash.Value = objectState.HyperDash.Value;
+            HitObject = entry.HitObject;
+
+            AccentColour.Value = entry.AccentColour;
+            HyperDash.Value = entry.HyperDash;
+
+            Scale = Vector2.Divide(entry.DisplaySize, Size) * 0.5f;
```

Comment:
Where is this `0.5f` coming from? I'm not sure I saw it be there before.

Reply:
It was `caughtObject.Scale /= 2` <https://github.com/ppy/osu/pull/11255/files#diff-c3ab09080a53b7f21d7ceffeaf2e5bfd4ab7ce5c5262696ea8d725197f87c640L476>.